clients.asciidoc

```diff
@@ -75,28 +75,28 @@ Advantages:
 
 Disadvantages:
 
-* No other users means that it doesn't behave the same as a public blockchain. There's no competition for transaction space or sequencing of transactions.
+* Having no other users means that it doesn't behave the same as a public blockchain. There's no competition for transaction space or sequencing of transactions
 * No miners other than you means that mining is more predictable, therefore you can't test some scenarios that occur on a public blockchain
-* No other contracts means you have to deploy everything that you want to test, including dependencies and contract libraries
+* Having no other contracts means you have to deploy everything that you want to test, including dependencies and contract libraries
 * You can't recreate some of the public contracts and their addresses to test some scenarios (e.g. the DAO contract)
 
 [[requirements]]
 == Hardware Requirements for a Full Node
 
-Before we get started, you should ensure you have a computer with sufficient resources to run an Ethereum full node. You will need at least 35 to 40GB of disk space to run a full node with a full copy of the Ethereum blockchain. If you want to also run a full node on the Ethereum testnet, you will need a minimum of 15GB of disk space for that. Downloading 50GB of blockchain will also require a fast Internet connection, otherwise it can take a long time.
+Before we get started, you should ensure you have a computer with sufficient resources to run an Ethereum full node. You will need at least 35 to 40GB of disk space to run a full node with a full copy of the Ethereum blockchain. If you want to also run a full node on the Ethereum testnet, you will need a minimum of 15GB of disk space for that. Downloading 50GB of blockchain will also require a fast Internet connection, as otherwise it can take a long time.
 
-Syncing the Ethereum blockchain is very input-output (I/O) intensive. It is best to have a Solid-State Drive (SSD). If you have a mechanical hard disk drive (HDD), you will need at least 8GB of RAM to use as cache, otherwise you may discover your system is too slow to keep up and sync fully.
+Syncing the Ethereum blockchain is very input-output (I/O) intensive. It is best to have a Solid-State Drive (SSD). If you have a mechanical hard disk drive (HDD), you will need at least 8GB of RAM to use as cache. Otherwise, you may discover your system is too slow to keep up and sync fully.
 
 Minimum Requirements:
 
 * CPU with 2 or 4 cores preferred
 * Solid State Drive (SSD) with at least 50GB free space
-* 4GB RAM minimum, 8GB+ if you have an HDD and not SSD.
+* 4GB RAM minimum, 8GB+ if you have an HDD and not SSD
```

Comment:
I removed the period mark here because most of the other entries didn't have one. Is there a preferred style for this sort of thing @aantonop?